apollo-api/src/main/java/com/apollographql/apollo/api/internal/json/JsonEncodingException.kt

```diff
@@ -13,13 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.apollographql.apollo.api.internal.json;
+package com.apollographql.apollo.api.internal.json
 
-import java.io.IOException;
-
-/** Thrown when the data being parsed is not encoded as valid JSON. */
-public final class JsonEncodingException extends IOException {
-  public JsonEncodingException(String message) {
-    super(message);
-  }
-}
+/**
+ * Thrown when the data being parsed is not encoded as valid JSON.
+ */
+class JsonEncodingException(message: String) : RuntimeException(message)
```

Comment:
What is the reason to make it a `RuntimeException` instead of an `IOException` like it used to be ?

Reply:
Are you suggesting to keep it as IOException but skip having @Throws annotation? I assume this turns checked exception to unchecked. In Kotlin the difference is thinner.

Reply:
Agh, it was my mistake. Will revert it back